### DISCUSSION TITLE
Errors Found in linked QR codes

### DISCUSSION THREAD

**Discussion by raghupathiguduri** [Sunbird-Knowlg/Community#67] [Q&A]
Hi Team, 
We are currently using the sunbird release-4.6.0 version. We are able to create the contents and publish but We have an issue while creating the collection with below scenario. 
1. Create Collection - After selecting the option 'QR code required' is 'Yes' and send for review, it shows the error message - `Errors found in linked QR Codes. Please check and correct`
 We tried to  debug this issue - but no error found in network inspect mode. No errors in dialcode service and QR code flink job.
 Attaching image for reference.
![All-Images-Awesome-Screenshot](https://user-images.githubusercontent.com/48198006/189484756-58f33478-b35e-488d-b70d-15fa10421ee5.png)

Kindly help us to identify the root cause and fix this issue. This is production environment issue.

**Comment by vaibhavbhuva** [comment]
@raghupathiguduri Looks look you have not linked the QR code to that particular node(folder) after selecting the option 'QR code required' is 'Yes'.  there is a facility to generate/download QR codes. So please generate and attached the same when you select the option 'QR code required' is 'Yes'. 
Otherwise, the editor will throw the same error as below: 
`Errors found in linked QR Codes. Please check and correct`  
And you will not be able to send it for review until you link the QR code to nodes that have selected the option 'QR code required' is 'Yes.

We are already showing the warning msg and indicating the unit node(folder) as red color.  
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/36467967/189903501-9a58f540-ebf5-4e8d-a8aa-a59ae381f554.png">

Another option is to select the option 'QR code required' as 'No'.  

cc: @vinukumar-vs

**Comment by komalm** [comment]
@raghupathiguduri - While editing a Public/Live course. We are facing the same QR code issue. Please check this Ticket for the same

https://github.com/orgs/Sunbird-Knowlg/discussions/69

cc @baljeetsingh
